cmd/kubectl-testkube/commands/testworkflows/run.go

```diff
@@ -78,6 +78,7 @@ func (e executionError) Unwrap() error {
 type RunOptions struct {
 	ExecutionName            string
 	Config                   map[string]string
+	Variables                []string
```

Comment:
shouldn't it be map?

Reply:
Depends what we like more:
1. With array we can do `--variable KEY1=value1 --variable KEY2=value` but we lose the automatic parsing of key=value
2. With map we can use only one flag like `--variable KEY1=value1,KEY2=value2`, but we get automatic parsing of key=value

I don't have a strong opinion, what do you think @vsukhin ?